.env

```diff
@@ -0,0 +1 @@
+ SKIP_PREFLIGHT_CHECK=true
```

Comment:
- You should put this file into .gitignore. and you can tell us in the comment while filling the form, what thing I should put in the .env file. Because it may contain the important API key and values which you don't want to share with all.

Reply:
This is issue was successfully addressed ✔️